src/bci_build/package/__init__.py

```diff
@@ -118,6 +118,51 @@ def __str__(self) -> str:
         return self.value
 
 
+@enum.unique
+class Registry(enum.Enum):
+    """Supported registries for container image distribution"""
+
+    SUSE_COM = "registry.suse.com"
+    APP_COLL = "dp.apps.rancher.io"
+    OPENSUSE_ORG = "registry.opensuse.org"
+
+    def __str__(self) -> str:
+        return self.value
+
+    @property
+    def application_prefix(self) -> str:
+        """Container image prefix for application containers."""
+        match self.value:
+            case Registry.SUSE_COM.value:
+                return "suse"
+            case Registry.APP_COLL.value:
+                return "containers"
+            case Registry.OPENSUSE_ORG.value:
+                return "opensuse"
+
+    @property
+    def development_prefix(self) -> str:
+        """Container image prefix for development containers."""
+        match self.value:
+            case Registry.SUSE_COM.value:
+                return "bci"
+            case Registry.APP_COLL.value:
+                return "containers"
+            case Registry.OPENSUSE_ORG.value:
+                return "opensuse/bci"
+
+    @staticmethod
+    def app_coll_title(title_property):
+        """Decorator that modifies the"""
```

Comment:
nit1: app_coll_title? what does this mean? app_collection_title or registry_title is more meaningful.
nit2: modifies what?